flowzone.yml

```diff
@@ -4,13 +4,20 @@
     uses: actions/download-artifact@v3
     with:
       name: source-${{ github.event.pull_request.head.sha || github.event.head_commit.id }}
-      path: /tmp
+      path: ${{ runner.temp }}
 
   - &extractSourceArtifact
     name: Extract source artifact
+    if: runner.os != 'Windows'
     working-directory: .
-    shell: bash
-    run: tar -xvf /tmp/source.tgz
+    run: tar -xvf ${{ runner.temp }}/source.tgz
+
+  - &extractSourceArtifactWindows
+    name: Extract source artifact
+    if: runner.os == 'Windows'
+    shell: powershell
```

Comment:
Is explicitly choosing powershell required to run on windows?

Reply:
Yeah, there was a reason for it. I think it was to do with tar.

Reply:
@Page- I am hesitant to go through retesting it all from the top now having spent all that time tuning the Etcher build... I am fairly certain I had to split these jobs or a reason.

Reply:
The main reason is it'd let us drop a bunch of code if we can use the same block for both, and it looks like that should be fine since the default shell for windows is powershell anyway: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell
Although it'd also be possible to specify `bash` and that should still work fine on windows, or `pwsh` and that should also work fine on linux

Reply:
tar doesn't work when running on Windows and with bash as the shell.

Reply:
Try `shell: pwsh`